bins/ggdeploymentd/src/deployment_handler.c

```diff
@@ -1875,7 +1877,13 @@ static GglError add_arn_list_to_config(
 
     // add configuration arn to the config if it is not already present
     // added to the config as a list, this is later used in fss
-    GglBuffer arn_list_mem = GGL_BUF((uint8_t[128 * 10]) { 0 });
+    GglBuffer arn_list_mem = GGL_BUF((
+        uint8_t
+            [(unsigned long) DEPLOYMENT_TARGET_NAME_MAX_CHARS *(unsigned long
+             ) MAX_DEPLOYMENT_TARGETS
+             + sizeof(GglList)
```

Comment:
Why sizeof list?

Reply:
I believe we need to deserialize the list, which includes allocating the list, the objects, and the buffers for each arn

Reply:
Removed because the passed list object is the memory for the list, so it doesn't need to be allocated